client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -62,14 +64,15 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
 
 				<Card className="steps__form">
-					<form>
+					<form onSubmit={ this.handleSubmit }>
 						<FormFieldset>
 							<FormLabel htmlFor="title">{ translate( 'Site Title' ) }</FormLabel>
 							<FormTextInput
 								autoFocus
 								id="title"
 								onChange={ this.setTitle }
 								value={ this.state.title }
+								required
```

Comment:
I thought we're alphabetizing props too? 🔤 :trollface:

Reply:
oooops 👍